docs/basic_concepts.rst

```diff
@@ -0,0 +1,172 @@
+Basic Concepts
+==============
+
+Before diving into the API, it's important to understand the fundamental
+data types of IOTA.
+
+The `official IOTA documentation site`_ gives a good and in-depth explanation of the
+concepts used in IOTA. PyOTA documentation will try to give references to the
+official site wherever possible.
+
+Ternary
+-------
+
+IOTA uses the `ternary numerical system`_ to represent data. The smallest unit of
+information is a ``trit``, that can have a value of -1, 0 or 1 in a balanced
+ternary system. A combination of 3 ``trits`` equals one ``tryte``, therefore a
+``tryte`` can have 3 * 3 * 3 = 27 different values.
+
+To represent a ``tryte``, IOTA encodes these 27 values into characters based on
+the `tryte alphabet`_.
+
+.. py:currentmodule:: iota
+
+In PyOTA, ``trits`` are represented as a sequence of numerical values (List[``int``])
+while trytes have their own class called :py:class:`TryteString`.
+
+IOTA token
+----------
+
+The `IOTA token`_ is a `unit`_ of value that can be transferred over an IOTA
+network through `transfer bundles`_.
+
+The IOTA token was launched on the Mainnet in June 2017. At this point, the
+nodes in the network were hard-coded with a total supply of 2,779,530,283 277,761.
+This large supply allows each of the billions of devices, which are expected to
+be a part of the Internet of Things, to have its own wallet and transact
+micropayments with other devices.
+
+Seed
+----
+
+`Seed`_ in IOTA is your unique password. It is the digital key that unlocks
+your safe that holds your tokens, or proves the ownership of messages.
+
+Seeds in PyOTA are always 81 trytes long and may only contain characters from
+the `tryte alphabet`_.
+
+.. warning::
+    Treat your seed(s) the same as you would the password for any other financial service.
+    Never share your seed with anyone and never use online generators to create a seed.
+    The library can help you to create your own locally and it does not require
+    internet connection: :py:meth:`iota.crypto.Seed.random`.
+
+For PyOTA specific implementation details on seeds, see :py:class:`crypto.Seed`.
+
+Address
+-------
+
+To send or receive any transaction (let them be zero-value or value transacitons)
+in IOTA, you will need to specify an `address`_. An address is like a physical mailbox
+on your entrance door: anyone can drop things in your mailbox (send you messages
+or tokens), but only you can empty it (withdraw tokens).
+
+.. warning::
+    Addresses should not be re-used once they are `spent from`_.
+    **You can receive as many transactions to an address as you wish, but only
+    spend from that address once.**
+
+    That being said, if you received money in your mailbox, you better move to a
+    new flat.
```

Comment:
❔ I read this as using metaphor to point out how ridiculous the restriction is (possibly not the way it was intended), which may be counter-productive to the message of the warning.

Reply:
Thanks for pointing this out! Gonna remove it then.